packages/components/src/components/tooltip/_tooltip.scss

```diff
@@ -545,6 +545,7 @@
 
   .#{$prefix}--tooltip--shown {
     display: block;
+    margin-top: $spacing-03;
```

Comment:
I think we'll need to add these changes to `src/globals/scss/_tooltip.scss` to account for all the alignment variants